bin/ssh_scan

```diff
@@ -65,10 +65,12 @@ if options[:target].nil?
   exit
 end
 
-unless options[:target].ip_addr? || options[:target].fqdn?
-  puts opt_parser.help
-  puts "\nReason: #{options[:target]} is not a valid target"
-  exit
+for ip in options[:target]
+  unless ip.ip_addr? || ip.fqdn?
+    puts opt_parser.help
+    puts "\nReason: #{options[:target]} is not a valid target"
+    exit
+  end
```

Comment:
for loops in Ruby are not preferred based on the style guide (https://github.com/bbatsov/ruby-style-guide)

A more preferred way to express this would be like so...

    options[:target].each do |target|
      unless target.ip_addr? || target.fqdn?
        puts opt_parser.help
        puts "\nReason: #{options[:target]} is not a valid target" 
        exit
      end
    end